src/ce.js

```diff
@@ -40,8 +40,9 @@ var ce = {
             'tag_name': elem.tagName
         };
 
-        if (_.includes(['input', 'select', 'textarea'], elem.tagName.toLowerCase())) {
-            props['value'] = this._getFormFieldValue(elem);
+        var formFieldValue = this._getFormFieldValue(elem);
```

Comment:
instead of returning null from `_getFormFieldValue`, why not check `_includeProperty` first here?